elements/pfe-navigation/demo/index.html

```diff
@@ -0,0 +1,414 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+    <title>PatternFly Element | Navigation Demo</title>
+
+    <link rel="stylesheet" href="../../pfe-layouts/pfe-base.css" />
+    <!-- <link rel="stylesheet" href="../../pfe-layouts/pfe-context.css" /> -->
+    <link rel="stylesheet" href="../../pfe-layouts/pfe-layouts.css" />
+
+    <noscript>
+      <link href="../../pfelement/pfelement-noscript.min.css" rel="stylesheet">
+    </noscript>
+
+    <link href="../../pfelement/pfelement.min.css" rel="stylesheet">
+    <link href="../pfe-navigation--lightdom.css" rel="stylesheet">
+    <link href="demo.css" rel="stylesheet" />
+
+    <!-- uncomment the es5-adapter if you're using the umd version -->
+    <script src="/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>
+    <script src="/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+    <script src="/requirejs/require.js"></script>
+
+    <script>require([
+      '../pfe-navigation.umd.js',
+      '../../pfe-band/pfe-band.umd.js',
+      '../../pfe-card/pfe-card.umd.js',
+      '../../pfe-cta/pfe-cta.umd.js'
+    ])</script>
+
+    <!-- <script type="module" src="../pfe-navigation.js"></script> -->
+  </head>
+
+  <body unresolved>
+    <h1>&lt;pfe-navigation&gt;</h1>
+    <pfe-navigation pfe-sticky pfe-close-on-click="external">
+      <!-- Skip navigation -->
+      <div slot="skip">
+        <ul>
+          <li><a href="#rh-main-content">Skip to content</a></li>
+        </ul>
+      </div>
+      <!-- Site logo -->
+      <h2 slot="logo">
+        <a href="https://redhat.com">
+          <img class="redhat-logo" src="https://via.placeholder.com/150x50.png" class="screen" title="Red Hat"/>
```

Comment:
The `class` attribute is duplicated on this line.

Reply:
Looks like this is outdated now